### ISSUE TITLE
Feature Request: Muscle Memory Resistance

### ISSUE THREAD

**Issue by inertia186** [MattyIce/steem-keychain#142]
I have joined d.tube's community twice now because I accidentally authorized `["subscribe",{"community":"hive-196037"}]`.  The [d.tube site](https://d.tube) requests this immediately after the page loads.

Of course, I should most certainly pay more attention to what Steem Keychain is saying, and I take full responsibility for my own oversight.  But I also think it would be an excellent feature if Steem Keychain could be configured to have a confirm count-down to disable the button for a few seconds.  Many apps implement this idea to improve security.  Like when you install a Google Extension, it won't let you click on the button right away.

In addition, maybe it would be a good idea to restrict Steem Keychain from broadcasting anything if the page has just loaded.

![image](https://user-images.githubusercontent.com/494368/75427205-e7aa7e00-58fa-11ea-895f-7c29134e258f.png)

**Comment by stoodkev** [comment]
Good idea, I could make this a setting so that it doesnt get too annoying for those that would put efficiency before security,
